writers/cassandra/messages.go

```diff
@@ -23,14 +29,27 @@ func New(session *gocql.Session) writers.MessageRepository {
 	return &cassandraRepository{session}
 }
 
-func (cr *cassandraRepository) Save(messages ...senml.Message) error {
+func (cr *cassandraRepository) Save(message interface{}) error {
+	switch m := message.(type) {
+	case mfjson.Messages:
+		return cr.saveJSON(m)
+	default:
+		return cr.saveSenml(m)
+	}
+}
+
+func (cr *cassandraRepository) saveSenml(messages interface{}) error {
+	msgs, ok := messages.([]senml.Message)
+	if !ok {
+		return errSaveMessage
+	}
 	cql := `INSERT INTO messages (id, channel, subtopic, publisher, protocol,
-			name, unit, value, string_value, bool_value, data_value, sum,
-			time, update_time)
-			VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+            name, unit, value, string_value, bool_value, data_value, sum,
```

Comment:
identation

Reply:
We're using spaces, it's probably just an issue with the preview.